hl7.py

```
@@ -60,12 +167,23 @@ async def start_tcp_server():
     async with server:
         await server.serve_forever()
 
-
+# Set the function run automatically when the application starts
 @app.on_event("startup")
```

Ruff (0.12.2)

Undefined name `app`

(F821)